HW_03/task2.py

```diff
@@ -0,0 +1,2 @@
+age_in_month =(2022-1977)*12+4
+#print(age_in_month)
```

Comment:
I think you wanted to uncomment here
```suggestion
print(age_in_month)
```